docs/troubleshooting-guide/datawarehouse.md

```diff
@@ -111,3 +111,6 @@ Metabase fails to connect to your data warehouse and the Metabase server logs in
 #### How to fix this
 
 Navigate to the options for your data warehouse and locate the Additional JDBC Connection Strings option, then add `trustServerCertificate=true` as an additional string.
+
+## I added tables or fields to my database but don't see them in Metabase
+If you have added new keys in a MongoDB document but Metabase does not reflect this change after sync, then it might be because Metabase hasn't seen that document for building its metadata. As MongoDB is a NoSQL database, documents in a single collection can vary enough so Metabase will not see all the variations of these to build the correct visualization of the fields. The solution for this issue is to include in the first document of the collection a sample document with all possible keys with null values to make Metabase recognize the correct schema of the entire collection, or modify any of the first documents of the collection to include all the possible keys that are included in the rest of the documents so Metabase can build the metadata correctly.
```

Comment:
The section is only about MongoDB and should probably get a title/subtitle indicating that. And `###` instead of `##`.
You cannot disable the sync process - for any database.
For other databases, it's likely because Metabase hasn't synced yet (hourly by default) or failed to sync (sometimes with errors in the log, other times for more technical reasons).